bioptim/dynamics/configure_problem.py

```diff
@@ -120,15 +120,15 @@ def torque_driven(
             A reference to the phase
         with_contact: bool
             If the dynamic with contact should be used
-        implicit_dynamics: bool
-            If the implicit dynamic should be used
+        multibody_dynamics: MultiBodyDynamics
+            which multibody dynamics should be used (explicit, implicit, semi_explicit)
         implicit_soft_contacts: bool
             If the implicit soft contact dynamic should be used
         fatigue: FatigueList
             A list of fatigue elements
         """
 
-        if implicit_dynamics:
+        if multibody_dynamics == MultiBodyDynamics.IMPLICIT:
```

Comment:
Is it an incapability or a not implemented yet? If it is the latter, please change the "ValueError" for a "NotImplementedYetError"